rbac/management/group/view.py

```diff
@@ -1126,14 +1141,41 @@ def remove_service_accounts(
 
             raise ValueError(f"Service account(s) {service_account_ids_diff} not found in the group '{group.name}'")
 
+        removed_service_accounts = []
         # Remove service accounts from the group.
         with transaction.atomic():
```

Comment:
I _think_ this additional atomic block is unnecessary now

Reply:
good catch, thanks!